ambari-server/src/main/java/org/apache/ambari/server/api/query/render/ClusterBlueprintRenderer.java

```diff
@@ -198,11 +199,10 @@ private Resource createBlueprintResource(TreeNode<Resource> clusterNode) {
     configProcessor.doUpdateForBlueprintExport();
 
     Set<StackId> stackIds = topology.getBlueprint().getStackIds();
-    if (stackIds.size() == 1) {
-      StackId stackId = Iterables.getOnlyElement(stackIds);
-      blueprintResource.setProperty("Blueprints/stack_name", stackId.getStackName());
-      blueprintResource.setProperty("Blueprints/stack_version", stackId.getStackVersion());
-    }
+    // TODO: mpacks should come from service groups once https://github.com/apache/ambari/pull/234 will be committed
+    Collection<Map<String, String>> mpackInstances = stackIds.stream().
```

Comment:
Will this query work for clusters that have been created by the Ambari UI?

Reply:
Yes, I am testing it with a cluster that has been created via UI.